<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_file"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAdd_curve"/>
   </widget>
   <widget class="QMenu" name="menuModify">
    <property name="title">
     <string>Modify</string>
    </property>
    <addaction name="actionModify"/>
   </widget>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_image"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuAdd"/>
   <addaction name="menuModify"/>
   <addaction name="menuSave"/>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionAdd_curve"/>
   <addaction name="actionModify"/>
   <addaction name="actionSave_Image"/>
   <addaction name="actionSave_file"/>
   <addaction name="actionOpen_file"/>
   <addaction name="actionOpen_image"/>
  </widget>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAdd_curve">
   <property name="text">
    <string>add curve</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>Select</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>open file</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen_image">
   <property name="text">
    <string>open image</string>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
